src/test/java/no/unit/nva/cristin/projects/FetchCristinProjectsTest.java

```diff
@@ -258,6 +273,51 @@ void handlerReturnsInternalErrorWhenUriCreationFails() throws Exception {
         assertTrue(gatewayResponse.getBody().contains(ERROR_MESSAGE_SERVER_ERROR));
     }
 
+    @Test
+    void handlerReturnsCristinProjectsWhenQueryContainsPageParameter() throws Exception {
+        InputStream input = requestWithQueryParameters(
+            Map.of(TITLE, RANDOM_TITLE, LANGUAGE_QUERY_PARAMETER, LANGUAGE_NB, PAGE, PAGE_NUMBER_TWO));
```

Comment:
Here you use the LANGUAGE_QUERY_PARAMETER  and before the LANGUAGE_KEY. any reason? Preferably, do not replicate the query parameters. use the constants defined in main